File: 0217-contains-duplicate/0217-contains-duplicate.cs
public class Solution {
    public bool ContainsDuplicate(int[] nums) {
        HashSet<int> hset = new HashSet<int>();
        foreach(int num in nums){
            if (hset.Contains(num)) { return true; }
            else hset.Add(num);
        }
        
        return false;
    }
}

File: 0217-contains-duplicate/0217-contains-duplicate.ts
function containsDuplicate(nums: number[]): boolean {
    const hset: Set<number> = new Set();
    for(let i:number = 0; i < nums.length; i++){
        if (hset.has(nums[i])){
            return true;
        }
        hset.add(nums[i]);
    }
    return false;
};